doc/source/cluster/cloud.rst

```diff
@@ -184,11 +184,55 @@ Private Cluster (List of nodes)
 
 The most preferable way to run a Ray cluster on a private cluster of hosts is via the Ray Cluster Launcher.
 
+.. tip:: To avoid getting the password prompt when running private clusters make sure to setup your ssh keys on the private cluster as follows:
+
+    .. code-block:: bash
+
+        $ ssh-keygen
+        $ cat ~/.ssh/id_rsa.pub >> ~/.ssh/authorized_keys
+
+Manually Managed
+~~~~~~~~~~~~~~~~
+
 You can get started by filling out the fields in the provided `ray/python/ray/autoscaler/local/example-full.yaml <https://github.com/ray-project/ray/tree/master/python/ray/autoscaler/local/example-full.yaml>`__.
 Be sure to specify the proper ``head_ip``, list of ``worker_ips``, and the ``ssh_user`` field.
 
 Test that it works by running the following commands from your local machine:
 
+.. code-block:: bash
+
+    # Create or update the cluster. When the command finishes, it will print
+    # out the command that can be used to get a remote shell into the head node.
+    $ ray up ray/python/ray/autoscaler/local/example-full.yaml
+
+    # Get a remote screen on the head node.
+    $ ray attach ray/python/ray/autoscaler/local/example-full.yaml
+    $ # Try running a Ray program with 'ray.init(address="auto")'.
+
+    # Tear down the cluster
+    $ ray down ray/python/ray/autoscaler/local/example-full.yaml
+
+Automatically Managed
+~~~~~~~~~~~~~~~~~~~~~
+
+Start by launching a coordinator server that will manage all the on prem clusters. The script for running the coordinator server is `ray/python/ray/autoscaler/local/coordinator_server.py <https://github.com/ray-project/ray/tree/master/python/ray/autoscaler/local/coordinator_server.py>`__. To launch the coordinator server run:
+
+.. code-block:: bash
+
+    $ python coordinator_server.py --ips <list_of_node_ips> --port <PORT>
```

Comment:
before getting into the docs themselves, please provide a "why" someone would be down this path. This can sit above (where my other comments are) but at the very least we need it somewhere. Basically, we need the simple reasons as to why someone must run `python coordinator_server.py`. 

Why is that necessary? Why do I want to do that? What benefits does that give me?

We have to answer all of those questions, and preferably in one place at the top where we have the definitions of "private cluster, automatically managed, etc."

Reply:
I guess I agree that these things should have been there originally too.